src/wallet/tests/test_api.py

```diff
@@ -73,44 +73,44 @@ def test_get_balance(self):
         self.assertEqual(balance1, balance2)
         # deposit 100 and mark as pending
         # pending and incoming transactions should not affect the balance
-        factories.WalletTrxFactory.create(
+        trx_obj = factories.WalletTrxFactory.create(
             wallet=wallet_obj,
-            trx_type=enums.TrxType.PENDING,
             amount=Money(100, wallet_obj.currency)
         )
+        factories.pend_trx(trx_obj)
```

Comment:
Will do. I think I worked this out in loops, the helper function is too similar to `set_status`.